biomage/experiment/utils.py

```diff
@@ -141,84 +141,19 @@ def get_cognito_username(email):
 
 def get_experiment_project_id(experiment_id, source_table):
 
-    table = boto3.resource('dynamodb').Table(source_table)
+    table = boto3.resource("dynamodb").Table(source_table)
 
     project_id = table.get_item(
-        Key={"experimentId": experiment_id},
-        ProjectionExpression='projectId'
-    ).get("Item")['projectId']
+        Key={"experimentId": experiment_id}, ProjectionExpression="projectId"
+    ).get("Item")["projectId"]
 
     return project_id
 
 
-def create_gem2s_hash(experiment, project, samples):
```

Comment:
`hashlib`, `collections`, `constants` now unused imports

Reply:
removed, I don't know why but the import auto-format and remove no longer works. It seems we did fuck it up in the end